### PR TITLE
Add Agra,India recommendations

### PR DESCRIPTION
**Pull request by GaurangSinghal**
This Pull Request closes issue #51.
### Adding restaurants and activities
- Does anyone have a favorite type of food ?
- Are you wanting outdoor activities or indoor activities ?

### CODE REVIEW

**Comment by marinafurman** [comment]
My New comment

File: Gaurang/Gaurang-India.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Agra india
+
```
Comment (jaitawatkuldeep) [inline Gaurang/Gaurang-India.md:2]: @GaurangSinghal Which place to visit in Agra

Comment (GaurangSinghal) [inline Gaurang/Gaurang-India.md]: Many places in here . Taj Mahal, Agra Fort, Sikandra Tomb, Fatehpur Sikri...

Comment (GaurangSinghal) [inline Gaurang/Gaurang-India.md]: @jaitawatkuldeep Where are you from ?

**Comment by GaurangSinghal** [comment]
C'mon

**Pull request merged**